wd_util.c

```diff
@@ -351,34 +352,72 @@ int wd_parse_async_poll_en(struct wd_env_config *config, const char *s)
 	return 0;
 }
 
-/* fix me */
-static int parse_ctx_num_on_numa(const char *s, int *ctx_num, int *node)
+static int parse_num_on_numa(const char *s, int *num, int *node)
```

Comment:
I have add a number check in this function

Reply:
The number check should be common. Is it right?